docs/environment_variables.md

```diff
@@ -79,3 +79,24 @@ SERVER_TRAFFIC_SECRET_0 XXXX
 CLIENT_HANDSHAKE_TRAFFIC_SECRET XXXX
 CLIENT_TRAFFIC_SECRET_0 XXXX
 ```
+
+`REQUESTS_CA_BUNDLE` `SSL_CERT_FILE` `CURL_CA_BUNDLE`
+-----------
+
+Valid values: a filename
+
+If one of these environment variables is set and `trust_env=True` and `verify=True` or `verify is None`, `verify` will be set by respectively `REQUESTS_CA_BUNDLE`, `SSL_CERT_FILE`, `CURL_CA_BUNDLE` environments as file.
+
+Example:
+
+```python
+# test_script.py
+
+import httpx
+client = httpx.Client(verify=None, trust_env=True)
+client.get("https://google.com")
+```
+
+```console
+REQUESTS_CA_BUNDLE=test.pem python test_script.py
```

Comment:
We should probably use `SSL_CERT_FILE` as the example, since it is [PEP 476](https://www.python.org/dev/peps/pep-0476/#trust-database).